changelog/issue-4624.md

```diff
@@ -0,0 +1,5 @@
+audience: users
+level: minor
+reference: issue 4624
+---
+The object service now supports an additional download method, `getUrl`, which handles gziped content and requires that hashes be validated.
```

Comment:
Can you add an additional comment here, that the client parts are not yet implemented, and will be provided in a future release?

Reply:
Yeah, that makes sense.